src/kernel/mconfig.ml

```diff
@@ -155,6 +157,12 @@ let dump_merlin x =
     "packages_path"    , `List (List.map ~f:Json.string x.packages_path);
 
     "failures"         , `List (List.map ~f:Json.string x.failures);
+    "assoc_suffixes"   , `List (
+      List.map ~f:(fun (suffix,reader) -> `Assoc [
+          "impl", `String suffix;
+          "intf", `String reader;
```

Comment:
c/p failure I guess, it should be `"reader"` not `"intf"`

Reply:
yep apologies - will fix